tests/basic/volume_ops_test.go

```diff
@@ -1484,3 +1488,135 @@ var _ = Describe("{ResizeVolumeAfterFull}", func() {
 	})
 
 })
+
+var _ = Describe("{CreateFastpathVolumeRebootNode}", func() {
+	var testrailID = 0
+	// JIRA ID : https://portworx.atlassian.net/browse/PTX-15700
+	var runID int
+	JustBeforeEach(func() {
+		StartTorpedoTest("CreateFastpathVolumeRebootNode",
+			"Create fast path volume, reboot the node, check fastpath is active", nil, testrailID)
+		runID = testrailuttils.AddRunsToMilestone(testrailID)
+		log.Infof("The runID  %v ", runID)
+	})
+
+	var pxNode node.Node
+	var contexts []*scheduler.Context
+	var applist = Inst().AppList
+	stepLog := "Create fastpath Volume Reboot Node and check if fastpath is active"
+	It(stepLog, func() {
+		log.InfoD(stepLog)
+		stepLog = "Step 1: Get all the Storage nodes and select a node for test"
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+			// Get all the Nodes
+			pxNodes, err := GetStorageNodes()
+			log.FailOnError(err, "Unable to get the storage nodes")
+
+			// Select random Storage node for the test
+			if len(pxNodes) > 0 {
+				pxNode = GetRandomNode(pxNodes)
+			} else {
+				log.FailOnError(errors.New("No Storage Node Availiable"), "Error occured while selecting StorageNode")
+			}
+
+			log.Infof("The Selected node for Fast path label is %v : ", pxNode.Name)
+
+			// Remove if node-type label is set before the test
+			Inst().AppList = applist
+			RemoveLabelsAllNodes(k8s.NodeType, true, false)
+		})
+
+		stepLog = "Step 2: Schedule application and Add label on the selected storage Node"
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+			var err error
+
+			// Add label on the selected node
+			err = Inst().S.AddLabelOnNode(pxNode, k8s.NodeType, k8s.FastpathNodeType)
+			log.FailOnError(err, fmt.Sprintf("Failed add label on node %s", pxNode.Name))
+			Inst().AppList = []string{"fio-fastpath-repl1"}
```

Comment:
make sure it is reverted to initial list at the end of the test

Reply:
Fixed